app/views/greetings/random_greeting.html.erb

```diff
@@ -0,0 +1,2 @@
+    <div id="root"></div>
```

Comment:
**OPTIONAL** - You have done well so far, but after creating your root route and a controller with one action, it should also have a corresponding view, Please put this `<div id="root"></div>` in the corresponding action view for your root route to ensure your app works properly 🙏